### ISSUE TITLE
It needs to be downloaded every time and it's very slow

### ISSUE THREAD

**Issue by 11242294** [23Aaron/Betelguese#3]
It needs to be downloaded every time and it's very slow

**Comment by JaiGupta13** [comment]
Agree, a way to cache the required downloads files would be nice however I doubt anyone would need to run the app many times.

**Comment by NSAntoine** [comment]
I defintely agree, it should be cached

**Comment by kirb** [comment]
Not fully fixed, it still redownloads files every time, but this has been improved in 1.1 as it’ll only download the files specific to your iOS version, rather than downloading all of the files and only using some of them.
